src/c++/perf_analyzer/genai-pa/genai_pa/wrapper.py

```diff
@@ -55,10 +55,9 @@ def build_cmd(model, args, extra_args):
             "input_type",
             "input_format",
             "output_format",
+            # The 'streaming' passed in to this script is to determine if the LLM response should be streaming. That is different than the 'streaming' that PA takes, which means something else (and is required for decoupled models into triton)
```

Comment:
can we make the comment less than 80 characters per line?

Reply:
Yeah, Elias and I have been trying to figure out how to do that :P  Black doesn't seem to care. I'm going to manually chop it up.